daemon/containerd/mount.go

```diff
@@ -0,0 +1,51 @@
+package containerd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/containerd/containerd/mount"
+	"github.com/docker/docker/container"
+	"github.com/sirupsen/logrus"
+)
+
+// Mount mounts and sets container base filesystem
+func (i *ImageService) Mount(ctx context.Context, container *container.Container) error {
+	snapshotter := i.client.SnapshotService(i.snapshotter)
+	mounts, err := snapshotter.Mounts(ctx, container.ID)
+	if err != nil {
+		return err
+	}
+
+	tempMountLocation := os.TempDir()
+
+	root, err := os.MkdirTemp(tempMountLocation, "rootfs-mount")
```

Comment:
Should this path be scoped or at least recognizable based on some aspect of the container like the container ID? :sweat_smile:

Reply:
Great catch, will do

Reply:
I gave it a little nudge to include the container ID 😅